core/src/com/po/fuck/view/Sprites/BasicSpriteInfo.java

```diff
@@ -0,0 +1,54 @@
+package com.po.fuck.view.Sprites;
+
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.utils.JsonValue;
+
+/*
+ * assetsData.json description
+ * all textures have:
+ * each .png stores some table of frames for the object
+ * path - path to texture
+ * frameWidth - the width of single frame
+ * frameHeight - the height of single frame
+ * width - the total width of this texture
+ * height - the total height of this texture
+ * frameDuration - the time between frames in seconds
+ */
+
+public class BasicSpriteInfo {
+    public String path;
+    public Vector2 frameSize;
+
+    public double frameDuration;
```

Comment:
Why is it public, but also has getters ?
Do we need to have a `path` here ?

Reply:
1. Will fix
2. Yes